### DISCUSSION TITLE
Check/Validate an error job

### DISCUSSION THREAD

**Discussion by Oli66178** [jhuckaby/Cronicle#800] [Q&A] [answered]
Hi.
Is there a way to check/validate a job terminated with error status ?
So others users know that someone have verified this error status.

Many thank's.

**Comment by jhuckaby** [comment]
Can you provide more information please?  I do not understand the question.

You can see if a job terminated with error status right in the UI, on the [Job Details Tab](https://github.com/jhuckaby/Cronicle/blob/master/docs/WebUI.md#job-details-tab):

![Screenshot](https://pixlcore.com/screenshots/Screenshot_2024-08-28_at_9.35.41_AM.png)

You can see which jobs were errors on the [Event History Tab](https://github.com/jhuckaby/Cronicle/blob/master/docs/WebUI.md#event-history-tab):

![Screenshot](https://pixlcore.com/screenshots/Screenshot_2024-08-28_at_9.36.06_AM.png)

You can also use the [get_job_status JSON API](https://github.com/jhuckaby/Cronicle/blob/master/docs/APIReference.md#get_job_status) to fetch information about a completed job to see if it completed successfully or failed (see the `code` property).

Does this help?

**Reply by Oli66178** [reply]
Thank you for theses details but :
Is it possible to change the status of an error job from "error" to "error but verified by ..." ?
And perhaps also change the color status from red to blue ?
And if we could put a comment on this verification, this would be perfect ;-)
So other members of the supervising team know that someone have already checked this error and then, the global status of Cronicle is OK.

I hope I've made my explanation clearer.

**Comment by dheise-rheinenergie-trading-com** [comment]
Sounds like a really great idea. I would also be in favor of such a feature.

**Comment by jhuckaby** [comment, marked as answer]
I completely understand now.  Thank you for the explanation.  Awesome idea, and I've actually already implemented this, or rather something very close to it, in the upcoming Cronicle v2...

In Cronicle v2 there is a concept of "tags", and these are completely user-defined.  You can create as many tags as you like, and give them custom names and custom icons:

![Screenshot 2024-08-29 at 9 23 23 AM](https://github.com/user-attachments/assets/230e7df0-1c81-43d4-9e40-c534d5d8ded1)

So in your case you could add a tag called "**Verified**" and pick a special icon for it:

![Screenshot 2024-08-29 at 9 23 12 AM](https://github.com/user-attachments/assets/94c0aa12-d8fc-4f37-aec5-f37c4439cbca)

![Screenshot 2024-08-29 at 9 22 45 AM](https://github.com/user-attachments/assets/d6e48127-05aa-4a58-83c8-c10f23764d7e)

You can assign tags to events, so they apply to every job for that event, or you can apply them DURING a job inside the running job code itself:

```sh
echo '{ "orchestra": true, "push": { "tags": ["flag"] } }'
```

Or, and this is probably what you really want: you can apply tags to a completed job right in the UI:

![Screenshot 2024-08-29 at 9 26 01 AM](https://github.com/user-attachments/assets/e412dbe7-727c-4618-9b17-13a2d8c43329)

So in your case, when a team member comes in and looks at a failed job, they can add the "Verified" tag to the job!

And then later you can also search for jobs by tag:

![Screenshot 2024-08-29 at 9 29 01 AM](https://github.com/user-attachments/assets/7903baaf-1665-4c9b-9bf6-6972ee42f35b)

Would this work for you?

This is coming in v2 which is due for release in 6 months or so.  I'm targeting early 2025 (sorry, the target date keeps moving, but I am working extremely hard on it, every single day).

**Comment by Oli66178** [comment]
It seems to be very good.
It would be perfect if we could add a comment to explain/justify the error.
A simple text field on each job would be sufficient.

Many thank's for this great job !
